packages/relay/src/routes/reportsRoute.ts

```diff
@@ -0,0 +1,20 @@
+import { Express } from 'express'
+import { DB } from 'anondb'
+import { reportService } from '../services/ReportService'
+import { errorHandler } from '../services/utils/ErrorHandler'
+
+export default (app: Express, db: DB) => {
+    app.post(
+        '/api/reports/claimPositiveReputation',
```

Comment:
I suggest to use `/api/reputation/claim` as the url prefix in order to put related logic together.

Reply:
fixed